src/main/java/ch/njol/skript/lang/function/ScriptFunction.java

```diff
@@ -31,6 +28,7 @@
 import ch.njol.skript.lang.util.SimpleEvent;
 import ch.njol.skript.variables.Variables;
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.eclipse.jdt.annotation.Nullable;

```

Comment:
Do not change import order. It will mess up organizing imports with Eclipse in future.